DigitalLearningSolutions.Data/Models/User/UserEntity.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace DigitalLearningSolutions.Data.Models.User
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class UserEntity
+    {
+        private const int FailedLoginThreshold = 5;
+
+        public UserEntity(
+            UserAccount userAccount,
+            IEnumerable<AdminAccount> adminAccounts,
+            IEnumerable<DelegateAccount> delegateAccounts
+        )
+        {
+            UserAccount = userAccount;
+            AdminAccounts = adminAccounts;
+            DelegateAccounts = delegateAccounts;
+        }
+
+        public UserAccount UserAccount { get; set; }
+        public IEnumerable<AdminAccount> AdminAccounts { get; set; }
+        public IEnumerable<DelegateAccount> DelegateAccounts { get; set; }
+
+        public bool AdminAccountsLocked => UserAccount.FailedLoginCount >= FailedLoginThreshold;

```

Comment:
Sorry, this name isn't quite as sensible as it was when we were only locking admin accounts.
Probably more useful to have an `IsLocked` property `= UserAccount.FailedLoginCount >= FailedLoginThreshold && AdminAccounts.Any()`.